### DISCUSSION TITLE
How to Access the developer pack , can anyone help me out

### DISCUSSION THREAD

**Discussion by DeveloperRVerma** [github/india#427] [GitHub Platform]
How to Access the developer pack , can anyone help me out

**Comment by TheSuranaverse** [comment]
@DeveloperRVerma I will guide & assist you step by step on how I & the other 173 students also got the pack who were facing the same issue as you. So I would request you to please join the link below & let me know once you have joined, please.

Link: https://discord.gg/drQ7ySCdX2

Also please mark this reply as answered once you have joined 😊

**Reply by DeveloperRVerma** [reply]
@Neklaustares-tPtwP  I have joined the discord platform , so please clear my doubts in my discussions.

**Comment by FRTMentor** [comment]
From FRT we are not supporting the activation of the Developer pack.
Please reach out the GitHub support team for more information.
